packages/localizations/src/en-US.ts

```diff
@@ -135,6 +135,19 @@ export const enUS: LocalizationResource = {
       subtitle: 'to continue to {{applicationName}}',
       actionLink: 'Use another method',
     },
+    forgotPassword: {
+      title_email: 'Check your email',
+      title_phone: 'Check your phone',
+      subtitle: 'to reset your password',
+      formTitle: 'Reset password code',
+      formSubtitle: 'Enter the code sent to your email address',
+      resendButton: 'Resend code',
+    },
+    resetPassword: {
+      title: 'Reset Password',
+      formButtonPrimary: 'Reset Password',
+      successMessage: 'Your password was successfully changes. Now signing you in, please wait a moment.',
```

Comment:
changed* but probably "reset" or "set" is better. After that maybe just "Signing you in..." ?